src/main/java/pl/net/was/OpenApiClient.java

```diff
@@ -486,14 +487,17 @@ private Iterable<List<?>> convertJson(OpenApiTableHandle table, JsonNode jsonNod
     {
         ImmutableList.Builder<List<?>> resultRecordsBuilder = ImmutableList.builder();
 
-        Map<String, Object> pathParams = getFilterValues(table, PathItem.HttpMethod.GET, null);
+        Map<String, Object> params = new HashMap<>();
+        params.putAll(getFilterValues(table, PathItem.HttpMethod.POST, null));
```

Comment:
We should pass which method is being used to `convertJson`

Reply:
Good point

Reply:
Done :)